courseaffils/models.py

```diff
@@ -141,7 +141,9 @@ class CourseInfo(models.Model):
     # for ability to query what courses are going on NOW
     starttime = models.TimeField(null=True, blank=True)
     endtime = models.TimeField(null=True, blank=True)
-    days = models.CharField(max_length=7, null=True, blank=True)  # e.g. 'MWF'
+    days = models.CharField(
+        max_length=7, null=True, blank=True,
+        help_text='e.g. "MTWRF"')
```

Comment:
IIRC, you're going to need to make a migration to go along with this. It doesn't change anything in the db schema, but django likes to have migrations even when you change the `help_text` and it will end up complaining and doing weird things (complaining that you need to run 'makemigrations' and then doing nothing when you do, etc).